docs.openc3.com/docs/configuration/interfaces.md

```diff
@@ -188,3 +188,7 @@ PROTOCOL WRITE CrcProtocol CRC # See the documentation for parameters
 Note the first parameter after the PROTOCOL keyword is how to apply the protocol: READ, WRITE, or READ_WRITE. Read applies the protocol on incoming packets (telemetry) and write on outgoing packets (commands). The next parameter is the protocol filename or class name. All other parameters are protocol specific.
 
 In addition, you can define your own protocols which are declared like the COSMOS helper protocols after your interface. See the [Custom Protocols](../configuration/protocols.md#custom-protocols) documentation for more information.
+
+:::info Protocol Run Order
+Read protocols execute in the order specified (First specified runs first). Write protocols execute in the reverse order (Last specified executes first). This allows READ_WRITE protocols to work as expected adding and removing layers in the correct corresponding orders.
```

Comment:
Last sentence is a little confusing and I think unnecessary. If you want to call out READ_WRITE then maybe:
"This also applies to READ_WRITE protocols which appends the protocol to the READ stack and prepends the protocol to the write stack."